apps/site/assets/css/_banner.scss

```diff
@@ -3,7 +3,7 @@
   display: block;
   margin-top: $base-spacing * 3;
 
-  &:hover {
+  &:hover, &:focus {
```

Comment:
I believe each selector should be on its own line, but that might depend on the linter we are using for SCSS.